stripe-ui-core/api/stripe-ui-core.api

```diff
@@ -126,6 +126,8 @@ public final class com/stripe/android/uicore/elements/IdentifierSpec$$serializer
 
 public final class com/stripe/android/uicore/elements/IdentifierSpec$Companion {
 	public final fun Generic (Ljava/lang/String;)Lcom/stripe/android/uicore/elements/IdentifierSpec;
+	public final fun getBlik ()Lcom/stripe/android/uicore/elements/IdentifierSpec;
+	public final fun getBlikCode ()Lcom/stripe/android/uicore/elements/IdentifierSpec;
```

Comment:
Can we mark these with `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`?

For context: The entire `companion object` should have been marked with it, but I guess the original author forgot. Let’s at least make sure that new fields don’t leak.

Reply:
```
        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
        val Blik = IdentifierSpec("blik", requestDestination = RequestDestination.Options)

        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
        val BlikCode = IdentifierSpec("blik[code]", requestDestination = RequestDestination.Options)   
```

I've added the `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` but apiDump and apiCheck don't seem to care and keep trying to add them back or failing

Reply:
Hmm, not sure about it then 🤔 Feel free to leave as-is and we can take care of that later.